### ISSUE TITLE
Can't have simultaneous connections

### ISSUE THREAD

**Issue by zgababa** [ericdraken/chrome-vpn#6]
Hi ! 

First of all, thanks for this amazing work ! 

Here my issue : 

I run this command  `docker-compose -f docker-compose-scale1.yaml up --scale chrome-vpn=6`

Everything seems fine : 
![image](https://user-images.githubusercontent.com/3789044/96186155-521c3700-0f3b-11eb-83d9-41cbd1bf87ab.png)

But when I hit this url `curl -x localhost:3001 ipinfo.io`
It's always the *same* server which respond to me. Did I miss something ? 

Btw when I hit `localhost:3001/status`
I received that, I don't know if it's linked.

![image](https://user-images.githubusercontent.com/3789044/96186474-c656da80-0f3b-11eb-90d2-ec618b014f0a.png)

Thanks ! 

Note : 
I always use the `ericdraken/chrome-vpn:latest` image in dockerfiles

**Comment by zgababa** [comment]
Ok for my second problem, I found the solution : 

![image](https://user-images.githubusercontent.com/3789044/96189514-e63ccd00-0f40-11eb-92a2-033d59a35106.png)

**Comment by zgababa** [comment]
Ok for my second problem I found the solution ` curl -x localhost:3001 http://localhost:8080/status` ;)

**Comment by zgababa** [comment]
Ok, my bad. I just noticed that you changed the way to scale. 
It's not `docker-compose -f docker-compose-scale.yaml up --scale chrome-vpn=3` but `docker-compose -f docker-compose-scale3.yaml up`. 

I was based on this old article https://ericdraken.com/headless-chrome-vpn-docker/

Btw why did you change that ? Is it still possible to have *six* simultaneous connections like before ? 

I tried to do a scale6 dockerfile, but I received only 4 different ips during my tests.

Thanks
